sentry/src/main/java/io/sentry/IScopes.java

```diff
@@ -414,21 +434,27 @@ default void configureScope(@NotNull ScopeCallback callback) {
    *
    * @return scope
    */
-  public @NotNull IScope getScope();
+  @ApiStatus.Internal
```

Comment:
Can't get rid of them because we use them in `Scopes` so they've been marked internal. We can always open them up if needed.